README.md

```diff
@@ -22,17 +24,37 @@ https://jsonschema.net/
 Prerequisite:
 
     pip install jsonschema
-    
+
 Run a validation:
 
     ./validate.sh
 
-### Lint JSON files
+### Build JS sources + TS definitions
+
+Prerequisite:
+
+    yarn install
+
+Build:
+
+    yarn build
+
+### Generate TS definitions out of the schemas
+
+Prerequisite:
+
+    yarn build:deps
+
+Generate:
+
+    yarn generate
+
+### Prettify TS & JSON files
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏Nice 🎉